src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/Microsoft.VisualStudio.ProjectSystem.Managed.VS.csproj

```diff
@@ -52,15 +52,6 @@
     <Compile Update="ProjectSystem\VS\PropertyPages\PropertyPage.cs">
       <SubType>UserControl</SubType>
     </Compile>
-    <Compile Update="ProjectSystem\VS\PropertyPages\PropertyPage.Designer.cs">
-      <DependentUpon>PropertyPage.cs</DependentUpon>
-      <SubType>UserControl</SubType>
-    </Compile>
-    <Compile Update="ProjectSystem\VS\PropertyPages\PropertyPageResources.Designer.cs">
-      <AutoGen>True</AutoGen>
-      <DesignTime>True</DesignTime>
-      <DependentUpon>PropertyPageResources.resx</DependentUpon>
-    </Compile>
```

Comment:
Were these `Update` items just not needed?

Reply:
Even though these files are checked into the repo, they are also generated. These updates should likely not be removed.

Reply:
No, it didn't look like it was needed, but I will re-add